MCprep_addon/util.py

```diff
@@ -557,6 +559,10 @@ def alphanum_key(key):
 
 	return sorted(elements, key=alphanum_key)
 
+def deprecation_warning():
+	print("DEPRECATION WARNING: THIS FUNCTION WILL BE DEPRECATED")
```

Comment:
Suggested docstring. Also, can we gate it so that it exits early (with no prints or anything) if it's in prod mode? This way we aren't polluting console logs for the next smaller couple releases of MCprep.

```suggestion
def deprecation_warning():
	"""Creates printouts in functions which will be removed on the v3.5 MCprep release"""
	print("DEPRECATION WARNING: This function will be deprecated in MCprep v3.5")
```